src/test/java/emissary/test/core/junit5/RegressionTestUtil.java

```diff
@@ -118,7 +118,7 @@ public static Path getTestResx() {
     }
 
     /**
-     * @see ExtractionTest#checkAnswers(Document, IBaseDataObject, List, String)
+     * See {@link ExtractionTest#checkAnswers(Document, IBaseDataObject, List, String)}
```

Comment:
This is only a marginally accurate reference.  Yes, it will get errorprone to stop complaining, but there's a fairly significant divergence in what's checked, especially the logged events.

Reply:
The reference could be easily removed if you see that as the best option. I agree it's only marginally accurate

Reply:
I'm OK with us just wiping this javadoc comment for now.  This is a plumbing method that's only used from a few locations,